iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/AbstractResourceCheck.java

```diff
@@ -72,4 +73,17 @@ public static void reportOnFalse(CheckContext ctx, Tree tree, String message) {
   public static void reportResource(CheckContext ctx, BlockTree resource, String message) {
     ctx.reportIssue(resource.labels().get(0), message);
   }
+
+  public static void reportOnDisabled(CheckContext ctx, BlockTree block, boolean enabledByDefault, String message) {
```

Comment:
Are there other already existing rules that can make use of this? As of now, this is only used in `DisabledLoggingCheck`, so we can just have it in there if there are no other existing rules.

Reply:
I will update the other checks in a later PR